### PR TITLE
test_containers_projects_summary.py fixes for CFME 5.7

### PR DESCRIPTION
**Pull request by pavelzag**
# Purpose or Intent

This PR fixes test_containers_projects_summary.py for new labels in CFME 5.7.
{{pytest: cfme/tests/containers/test_containers_projects_summary.py -v --use-provider container }}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit c3f1d1cf7d1b124f6c85ed73b6bbe8ba1d169661:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

File: cfme/tests/containers/test_containers_projects_summary.py [hunk 1]
```diff
@@ -14,11 +14,18 @@
...
+                                        'Pods', 'Nodes']
+projects_relationships_fields_57 = ['Containers Provider', 'Routes', 'Container Services',
+                                    'Replicators', 'Pods', 'Nodes']
+
+projects_relationships_fields_key = ({
+    version.LOWEST: projects_relationships_fields_lowest,
+    '5.7': projects_relationships_fields_57
+})
 
+projects_relationships_fields = version.pick(projects_relationships_fields_key)
```
Comment (jkrocil) [inline cfme/tests/containers/test_containers_projects_summary.py]: `version.pick` would be ran on module import - this is undesirable. Either do the version picking inside the test where you need it or if it's needed in multiple tests, define it as a function; example here: https://github.com/ManageIQ/integration_tests/blob/master/cfme/tests/infrastructure/test_host.py#L22

**Comment by dajoRH** [comment]
Lint report for commit 94303d40150e37fc3146bcf2a4bf9351dc3d9b1c:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
You have multiple commits in the PR. When you want your PR to be merged, please squash them. Do not squash them when working on the pull request as this allows to see the progress!

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 934751a9bcbc35e3ce3ec192ecb3b92009f15bf9:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit add3bcf28c607be42d1bda2321d7bc802933a783:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
You have multiple commits in the PR. When you want your PR to be merged, please squash them. Do not squash them when working on the pull request as this allows to see the progress!

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 2633dc85048b9b4745c0be179643fe0ce9778248:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit d7fcbc97007fa4b6829d0bb52230bb3882f65c41:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by pavelzag** [comment]
The failures here occur because Projects menus are empty. The tests are fine otherwise and should be run successfully. Please review and merge.

**Pull request merged**